sitestats/sitestats-impl/src/java/org/sakaiproject/sitestats/impl/report/ReportManagerImpl.java

```diff
@@ -119,9 +120,14 @@ public class ReportManagerImpl extends HibernateDaoSupport implements ReportMana
 	private ReportFormattedParams	formattedParams	= new ReportFormattedParamsImpl();
 
 	/** FOP */
-	private FopFactory				fopFactory		= FopFactory.newInstance();
+	private FopFactory				fopFactory;
 	private Templates				cachedXmlFoXSLT	= null;
 	private static final String		XML_FO_XSL_FILE	= "xmlReportToFo.xsl";
+	
+	{
+		// Initialize FopFactory with base URI
+		fopFactory = FopFactory.newInstance(new File(".").toURI());
+	}
```

Comment:
another improvement would be to move this to constructor initialization

Reply:
agree or static initialization